src/histogram_2d.ts

```diff
@@ -117,13 +117,15 @@ class Histogram2D implements IAppView {
 
   private toggleChangeType(changeType) {
     // console.log('changeType', changeType);
-    const cssClass = `.${changeType.type}-color`;
-
-    this.$ratio.selectAll(cssClass).classed('noColorClass', !changeType.isActive);
-    this.$histogramRows.selectAll(cssClass).classed('noColorClass', !changeType.isActive);
-    this.$histogramCols.selectAll(cssClass).classed('noColorClass', !changeType.isActive);
-
-    this.$node.selectAll(`div.ratio > .${changeType.type}`).classed('noColorClass', !changeType.isActive);
+    this.clearContent();
+    this.updateItems(this.selectedTables);
```

Comment:
This will trigger a reload of the same data every time the change type switches. Not really efficient.